bin/kafka-streams-application-reset.sh

```diff
@@ -18,4 +18,4 @@ if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
 
-exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.tools.StreamsResetter "$@"
+exec $(dirname $0)/kafka-run-class.sh kafka.tools.StreamsResetter "$@"
```

Comment:
I just used an existing package. I guess for `streams` the appropriate package name would be `org.apache.kafka.streams.tools`. Did not want to use this package name in `core` -- but if you think I should, no problem. @dguy @enothereska @miguno @guozhangwang